maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -1046,6 +1079,7 @@ class Orchestra(
             ) ?: throw MaestroException.ElementNotFound(
                 "Element not found: $description",
                 parentViewHierarchy.root,
+                DebugInfo("Element with $description not found. Check the UI hierarchy in debug artifacts to verify if the element exists.")
```

Comment:
can this include any helpful information? Eg if the selector is wrong, is there something on the screen with a similar name, perhaps?

Reply:
> Eg if the selector is wrong, is there something on the screen with a similar name, perhaps?

I thought about this but problem with this is we might be asking them to hide real regressions. What if accessibility text broke and we ask them to change, but on contrary its actually a regression 🤔

Reply:
How about:

```
Assertion '${command.condition.description()}' failed. 
Check the UI hierarchy in debug artifacts to verify the element state and properties.

Possible causes:
- Element selector may be incorrect - check if there are similar elements with slightly different names/properties.
- Element may be temporarily unavailable due to loading state
- This could be a real regression that needs to be addressed
```